src/AddEntry.php

```diff
@@ -72,6 +72,10 @@ public function __invoke(string $type, string $changelogFile, string $entry) : v
     /**
      * Locates the location within the changelog where the injection should occur.
      * Also determines if the injection is a replacement or an addition.
+     *
+     * @param array $contents
+     * @param string $type
+     * @return stdClass
```

Comment:
param and return annotations are not necessary when typehints exist, unless you add descriptions.

Reply:
True. Updating ...

Reply:
(I'll take care of these during merge.)

Reply:
Actually, this happened in other files too. My IDE automatically fill these ... Update them all?